packages/notification-center/src/emotion.d.ts

```diff
@@ -0,0 +1,11 @@
+import '@emotion/react';
+import type { MantineTheme } from '@mantine/core';
+
+declare module '@emotion/react' {
```

Comment:
mantine theme override

Reply:
Wondering if this override will erase some Mantine defaults we might be relying on them? 🤔

Reply:
This is the TS world, it doesn't affect the Mantine defaults.
It just extends the Mantine `theme` interface with what we've provided in the `theme` config object, so we will see that full interface when checking the `theme` object on the emotion styled components, like:
```
const SomeContainer = styled.div`
   color: ${({ theme }) => theme.colors.brand[0]};
`
```

Reply:
Ignore my previous comment. I didn't notice it was a type file. 😭